iotdb-client/client-py/iotdb/Session.py

```diff
@@ -1058,6 +1076,35 @@ def insert_aligned_tablets(self, tablet_lst):
                         self.connection_error_msg()
                     ) from None
 
+    def insert_relational_tablet(self, tablet):
+        """
+        insert one tablet, in a tablet, for each timestamp, the number of measurements is same
+            for example three records in the same device can form a tablet:
+                timestamps,     m1,    m2,     m3
+                         1,  125.3,  True,  text1
+                         2,  111.6, False,  text2
+                         3,  688.6,  True,  text3
+        :param tablet: a tablet specified above
+        """
```

Comment:
Better to explicitly indicate how devices are expressed in a relational Tablet (i.e., ID columns) in the example, which is quite different from the tree Tablet.

Reply:
fixed